translate/api/Tests/TestBatchTranslate.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Google.Cloud.Translate.V3.Samples;
+using GoogleCloudSamples;
+using Google.Cloud.Storage.V1;
+using Google.Apis.Storage.v1.Data;
+using System.Linq;
+
+namespace Tests
+{
+    public class BatchTranslateTest : IDisposable
+    {
+        protected string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        protected string _bucketName { get; private set; }
+        protected Bucket Bucket { get; private set; }
+        protected string InputUri { get; private set; } = "gs://cloud-samples-data/translation/glossary_ja.csv";
+        readonly CommandLineRunner _sample = new CommandLineRunner()
+        {
+            VoidMain = TranslateV3BatchTranslateTextMain.Main
+        };
+
+        // Setup
+        public BatchTranslateTest()
+        {
+            // Create temp bucket
+            var storageClient = StorageClient.Create();
+            _bucketName = "translate-v3-" + TestUtil.RandomName();
+            Bucket = storageClient.CreateBucket(ProjectId, _bucketName);
+        }
+
+        public void Dispose()
+        {
+            var storageClient = StorageClient.Create();
+
+            // Clean up output files.
+            var blobList = storageClient.ListObjects(_bucketName, "");
+            foreach (var outputFile in blobList.Where(x => x.Name.Contains("translation/")).Select(x => x.Name))
+            {
+                storageClient.DeleteObject(_bucketName, outputFile);
+            }
+            storageClient.DeleteBucket(_bucketName);
```

Comment:
You don't need to manually delete all the objects in the bucket. DeleteBucket receives DeleteBucketOptions (or similar name), one of those options is for also deleting the bucket contents when deleting the bucket.

Reply:
Could you illustrate more on that? or show me an example

Reply:
client.DeleteBucket(bucket, new DeleteBucketOptions { DeleteObjects = true });